tests/components/streams_bootstrap_v3/test_streams_app.py

```diff
@@ -143,13 +143,13 @@ def test_set_topics(self):
                         "example-input": {"type": "input"},
                         "b": {"type": "input"},
                         "a": {"type": "input"},
-                        "topic-extra2": {"role": "role2"},
-                        "topic-extra3": {"role": "role2"},
-                        "topic-extra": {"role": "role1"},
+                        "topic-extra2": {"label": "role2"},
+                        "topic-extra3": {"label": "role2"},
+                        "topic-extra": {"label": "role1"},
```

Comment:
should we rename the values as well?

Reply:
We can, but these are just some values that don't have any meaning. Could be `foo` or `bar` or `baz`